### DISCUSSION TITLE
Provide enumerable to auth languageCode

### DISCUSSION THREAD

**Discussion by KingDarBoja** [firebase/firebase-js-sdk#4835] [General]
Hello Firebase Team,

I have noticed that there hasn't been any documentation about the possible "string" values to pass into [languageCode](https://github.com/firebase/firebase-js-sdk/blob/d47fb414bfd9508f611a9adbc253fb360c88fbd3/docs-exp/auth.auth.md) and would be great if there were a enumerable (like the new auth Persistence) to have some insight of the supported languages by google.

Cheers!

**Comment by Feiyang1** [comment]
@sam-gc

**Reply by sam-gc** [reply]
Thanks for the feature request. Unfortunately I think it would be too difficult to create and maintain a set of valid entries for languageCode within the SDK, especially since that field [controls different systems](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#languagecode) which may not all have the same set of supported languages.

**Reply by KingDarBoja** [reply]
So there is no documentation to atleast find what are the possible languages being supported for this auth feature?

**Reply by sam-gc** [reply]
I'm not aware of any such list. I will chat with the team to see if we could publish one, but unfortunately for the time being there's nothing definitive.

**Reply by zacharymichaelmoore** [reply]
the language supported are here https://github.com/firebase/firebaseui-web/blob/master/LANGUAGES.md
